types/sector_size.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+// SectorSize is the amount of bytes in a sector. This amount will be slightly
+// greater than the number of user bytes which can be written to a sector due to
+// bit-padding.
+type SectorSize uint64
+
+const (
+	OneKiBSectorSize = SectorSize(iota)
+	TwoHundredFiftySixMiBSectorSize
```

Comment:
QuarterGiBSectorSize? SectorSize265MiB? Spelling out 256 seems overly verbose.

Reply:
Our names will get better once we figure out what the LCD of all current and future supported sector sizes. I promise.